### PR TITLE
Cleanup LoaderTest test

### PR DESCRIPTION
**Pull request by munkie**
Clean up LoaderTest
* Remove unused persister
* Remove unused local vars
* Use _setExpectedExcetion_ method for exception assertions

Also i would recommend to:
* Remove composer.lock it has outdated versions of dependencies.
* Use last stable version of phpunit (~4.8)

### CODE REVIEW

File: tests/Nelmio/Alice/Fixtures/LoaderTest.php [hunk 1]
```diff
@@ -94,11 +88,12 @@ public function testGetBadReference()
 
     public function testLoadUnparsableFile()
     {
-        try {
-            $res = $this->createLoader()->load($file = __DIR__.'/../support/fixtures/not-parsable');
-        } catch (\UnexpectedValueException $e) {
-            $this->assertEquals("{$file} cannot be parsed - no parser exists that can handle it.", $e->getMessage());
-        }
+        $file = __DIR__.'/../support/fixtures/not-parsable';
+        $this->setExpectedException(
```
Comment (tshelburne) [inline tests/Nelmio/Alice/Fixtures/LoaderTest.php]: Does our version of PHPUnit not have the support for annotated exceptions?

Comment (tshelburne) [inline tests/Nelmio/Alice/Fixtures/LoaderTest.php]: Actually, I see that we're using annotations elsewhere - is there any reason you're choosing not to here?

Comment (munkie) [inline tests/Nelmio/Alice/Fixtures/LoaderTest.php]: Because $file needs to be added to the beginning of expected exception message

Comment (tshelburne) [inline tests/Nelmio/Alice/Fixtures/LoaderTest.php]: Oh... wow, yea, good call. Attention to detail...

**Pull request merged**